docs/guides/quick-start/miranum-ide/create-process.md

```diff
@@ -0,0 +1,180 @@
+
+   | Define Input                                                         | Add Rules                                                      |
+   |----------------------------------------------------------------------|----------------------------------------------------------------|
+   | <img src={DmnDefineInput} alt="Define input" style={{width: 480}} /> | <img src={DmnAddRules} alt="Add rules" style={{width: 480}} /> |
+
+4. Define the output
+   * Double-click on `Then`
+   * Enter a label
+   * Enter the output expression (the variable that can be accessed in the process)
+   * Select the type of the output
+   * For every rule you added in step 3, add a corresponding output value
+
+   | Define Output                                                          | Add Output Values                                                             |
+   |------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+   | <img src={DmnDefineOutput} alt="Define output" style={{width: 480}} /> | <img src={DmnAddOutputValues} alt="Add output values" style={{width: 480}} /> |
+
+5. Select your `Hit Policy`
+
+   <img src={DmnSelectHitPolicy} alt="Select Hit Policy" style={{width: 480}} />
+
+
+### Link the Decision to your Process
+
+1. Open your BPMN diagram
+2. Select the `Business Rule Task`
+3. Expand `Implementation` in the Properties Panel
+4. Select `DMN` as the `Type`
+5. Enter the ID of your decision in the `Decision Ref` field
+
+   | `.dmn`                                                       | `.bpmn`                                                       |
+   |--------------------------------------------------------------|---------------------------------------------------------------|
+   | <img src={DmnKey} alt="Decision Key" style={{width: 480}} /> | <img src={BpmnKey} alt="Decision Key" style={{width: 480}} /> |
+ 
+6. Enter the variable name that will hold the output of the decision in the `Result Variable` field
+
+   | `.dmn`                                                             | `.bpmn`                                                             |
+   |--------------------------------------------------------------------|---------------------------------------------------------------------|
+   | <img src={DmnOutput} alt="Output variable" style={{width: 480}} /> | <img src={BpmnOutput} alt="Output variable" style={{width: 480}} /> |
+
+7. Add the input variable in the `Inputs` section
+
+   | `.dmn`                                                           | `.bpmn`                                                           |
+   |------------------------------------------------------------------|-------------------------------------------------------------------|
+   | <img src={DmnInput} alt="Input variable" style={{width: 480}} /> | <img src={BpmnInput} alt="Input variable" style={{width: 480}} /> |
+
```

Comment:
💭 - While reading over the Quickstart guide I was asking myself how I would deploy my processes to the Camunda Engine? Maybe we can add a section about this. 🤔

Reply:
Is there a showcase for demonstrating the deployment?